samples/nrf9160/asset_tracker/prj_nrf9160_pca20035ns.conf

```diff
@@ -0,0 +1,68 @@
+# General config
+CONFIG_NEWLIB_LIBC=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_ASSERT=y
+CONFIG_REBOOT=y
+CONFIG_DEBUG=y
+CONFIG_LOG_INPLACE_PROCESS=y
+
+# Trusted execution
+CONFIG_TRUSTED_EXECUTION_NONSECURE=y
+
+# Network
+CONFIG_NETWORKING=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_OFFLOAD=y
+
+# MQTT
+CONFIG_MQTT_SOCKET_LIB=y
+CONFIG_MQTT_LIB_TLS=y
+CONFIG_MQTT_MAX_PACKET_LENGTH=2048
+
+# LTE link control
+CONFIG_POWER_OPTIMIZATION_ENABLE=n
+CONFIG_LTE_LINK_CONTROL=y
+CONFIG_LTE_AUTO_INIT_AND_CONNECT=n
+
+# BSD library
+CONFIG_BSD_LIBRARY=y
+CONFIG_BSD_LIBRARY_TRACE_ENABLED=n
+
+# nRF Cloud
+CONFIG_NRF_CLOUD=y
+CONFIG_NRF_CLOUD_LOG=y
+CONFIG_NRF_CLOUD_LOG_LEVEL=4
+
+# Sensors
+CONFIG_SENSOR=y
+CONFIG_GPS_USE_SIM=y
+CONFIG_GPS_SIM_TRIGGER_TIMER_MSEC=30000
```

Comment:
Is it intentional to send data only every 30 seconds? If it's done to save power, it could be considered to disable logging as well

Reply:
This was done mainly to not use to much data.